app/src/main/scala/com/coralogix/operator/Main.scala

```diff
@@ -60,17 +57,17 @@ object Main extends App {
 
     val clients =
       logging.live ++ (operatorEnvironment >>>
-        (crd.live ++
-          serviceaccounts.live ++
-          clusterroles.live ++
-          clusterrolebindings.live ++
-          daemonsets.live ++
-          configmaps.live ++
-          pods.live ++
-
-          rulegroupsets.live ++
-          coralogixloggers.live ++
-          alertsets.live))
+        (CustomResourceDefinitions.live ++
```

Comment:
You know I never noticed these were named this way until now. Good change 😆

Reply:
Now every generated module matches the actual recommended ZIO module pattern, which I just learned yesterday that will be deprecated soon :D